<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>notificator_w_action</class>
 <widget class="QWidget" name="notificator_w_action">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">notificator_w_action {
    margin: 2px;
	border: 2px solid #ffe400;
	border-radius: 8px;
	background: black;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lbl_primary_text">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Primary Text</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_secondary_text">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string>Secondary Text</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_dismiss_timout">
     <property name="font">
      <font/>
     </property>
     <property name="text">
      <string>Сообщение закроется через %1 секунд</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
